parachain/pallets/bridge/src/channel/outbound.rs

```diff
@@ -39,9 +39,9 @@ impl<T: Config> OutboundChannel for BasicOutboundChannel<T> {
 	// This implementation is a WIP!
 	fn submit(&self, target: H160, payload: &[u8]) -> DispatchResult {
 		self.storage.try_mutate(|data| {
-			data.nonce += 1;
-			T::MessageCommitment::add(self.id, target, data.nonce, payload)?;
-			<Module<T>>::deposit_event(Event::MessageAccepted(self.id, data.nonce));
+			// data.nonce += 1;
+			// T::MessageCommitment::add(self.id, target, data.nonce, payload)?;
+			// <Module<T>>::deposit_event(Event::MessageAccepted(self.id, data.nonce));
```

Comment:
Are those commented out intentionally?

Reply:
This code is pending the decision whether to leave the bridge or not and which functionality to keep here exactly, since some changes are necessary for this code to work again. It's just WIP, in other words.